src/utils.spec.tsx

```diff
@@ -188,4 +206,38 @@ describe(__filename, () => {
       expect(getPathFromQueryString(history)).toEqual(null);
     });
   });
+
+  describe('getCodeLineAnchorGetter', () => {
+    it('returns getCodeLineAnchor if compareInfo is null', () => {
+      const _getCodeLineAnchor = jest.fn();
+      expect(
+        getCodeLineAnchorGetter({ _getCodeLineAnchor, compareInfo: null }),
+      ).toEqual(_getCodeLineAnchor);
```

Comment:
Can't this just be `getCodeLineAnchor`? I.E. the real function, not a mock? Since you aren't executing the function, I don't think you need to mock it.